test/test_mps.py

```diff
@@ -731,9 +731,6 @@ def mps_ops_modifier(ops):
         'nn.functional.interpolatearea': None,
         'nn.functional.interpolatebicubic': None,
         'nn.functional.interpolatetrilinear': None,
-        # TODO: max_pool2d for integral types fails the numerical test
-        'nn.functional.max_pool2d': (integral_types() if product_version < 14.0 else
-                                     [torch.int64, torch.int32, torch.int16, torch.int8]),
```

Comment:
Looks like you probably want to keep `integral_types() if product_version < 14.0 else []` as errors are coming from the MPS side on the older versions of MacOS